doc/dev/debugging.rst

```diff
@@ -388,3 +388,84 @@ corresponding arguments.
     $ /path/to/executable
     MemRef: base@ = 0x64fc9dd5ffc0 rank = 0 offset = 0 sizes = [] strides = [] data =
     25
+
+Mixed-mode debugging of Python and C++
+======================================
+
+Catalyst supports mixed-mode debugging of Python and/or C++ code when providing the ``debug_compiler=True`` flag to 
+the ``@qjit`` decorator. Enabling this option signals to the compiler to wait for an appropriate user-provided signal
+after launching the compiler process. Some notes about use of this support:
+
+* This functionality requires building Catalyst with debug symbols. This can be achieved via make all 
+  ``BUILD_TYPE="RelWithDebInfo"``. The debug symbols are only within available within the Catalyst-owned targets. 
```

Comment:
```suggestion
* This functionality requires building Catalyst with debug symbols. This can be achieved via
  ``make all BUILD_TYPE="RelWithDebInfo"``. The debug symbols are only within available
  within the Catalyst-owned targets. 
```